crates/rpc/src/method/get_events.rs

```diff
@@ -651,6 +659,9 @@ impl SerializeForVersion for EmittedEvent {
         serializer.serialize_optional("block_hash", self.block_hash)?;
         serializer.serialize_optional("block_number", self.block_number)?;
         serializer.serialize_field("transaction_hash", &self.transaction_hash)?;
+        if serializer.version >= RpcVersion::V10 {
+            serializer.serialize_field("transaction_index", &self.transaction_index.get())?;
```

Comment:
Aren't we missing `event_index` here?

Reply:
Ouch... :-) https://github.com/eqlabs/pathfinder/pull/3084/commits/dd2cd5ec20d61fb500785c1bd2273c411a000742